### DISCUSSION TITLE
How to fix theme group problem

### DISCUSSION THREAD

**Discussion by mahaveergurjar** [prasanthrangan/hyprdots#368] [General]
![231021_16h07m19s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/24ab07dd-beac-423e-8f3c-6adaae55d75d)

**Comment by mahaveergurjar** [comment]
and dunst are also break when change theme
![Uploading 231021_16h07m19s_screenshot.png…]()

**Comment by T-Crypt** [comment]
Check #327 and #332 closed issues go over this in depth to resolve. 

Install hyprland update and then make sure you have the latest dotfiles updated by running the install.sh script with the full script or using -r to restore configs 

Reboot will be required, just check out those 2 closed issues I posted the full resolution there.

**Comment by mahaveergurjar** [comment]
ok
